modin/test/interchange/dataframe_protocol/hdk/test_protocol.py

```diff
@@ -47,6 +48,17 @@ def test_simple_export(data_has_nulls, from_hdk, n_chunks):
     md_df = pd.DataFrame(data)
 
     exported_df = export_frame(md_df, from_hdk, n_chunks=n_chunks)
+
+    if StorageFormat.get() == "Hdk":
```

Comment:
How did it work before?

Why check the `StorageFormat` at all if these tests are specific to hdk?

Reply:
Before it was executed with arrow.

The check is not required here, thanks.

Reply:
Is there any chance to quickly fix this by defaulting to arrow?

Reply:
No. The main idea of this PR is to minimize the imports/exports from/to arrow. If a table has been imported into HDK, the subsequent execution should be performed on the HDK side.